more/pixels/pixelConfig.js

```diff
@@ -152,7 +151,7 @@ function makePixelConfigMaker(
         });
       },
       makeMintKeeper: makePixelMintKeeper,
-      strategy: makePixelStrategy(canvasSize),
+      strategyName: 'pixelStrategy10',
```

Comment:
I understand that hardcoding a 10 here is an awful kludge that we want to get away from as soon as possible. But as long as this PR does this hardcoding, it should also do an `insist` that the actual `canvasSize` is indeed 10